velox/exec/tests/RowContainerTest.cpp

```diff
@@ -187,6 +190,60 @@ class RowContainerTest : public exec::test::RowContainerTestBase {
     testRowNumbers();
   }
 
+  void testRowNumbersSpecialCases(
+      RowContainer& container,
+      const std::vector<char*>& rows,
+      int column,
+      const VectorPtr& expected) {
+    auto size = rows.size();
+    auto testRepeatedRowNumber = [&]() {
```

Comment:
Any reason not to extract this as a top-level method and call that instead of testRowNumbersSpecialCases?

Reply:
I extracted these two as top-level methods instead.